mentions-generator/src/main/java/de/brandwatch/minianalytics/mentiongenerator/MentionsGeneratorApplication.java

```diff
@@ -17,9 +17,11 @@ public static void main(String[] args) {
     @Autowired
     private Producer producer;
 
-    @PostConstruct
-    public void sendMessagesOnKafka() {
-        producer.send("Waz up?");
-    }
+    @Autowired
+    private Consumer consumer;
```

Comment:
Why are you autowiring all those things in this class?

Reply:
Removed